README.md

```diff
@@ -60,12 +60,12 @@ To print all go processes, run `gops` without arguments:
 
 ```sh
 $ gops
-983     uplink-soecks	(/usr/local/bin/uplink-soecks)
-52697   gops	(/Users/jbd/bin/gops)
-4132*   foops (/Users/jbd/bin/foops)
-51130   gocode	(/Users/jbd/bin/gocode)
+983   980    uplink-soecks go1.9 (/usr/local/bin/uplink-soecks)
+52697 52695  gops          go1.9 (/Users/jbd/bin/gops)
+4132  4130   foops*        go1.9 (/Users/jbd/bin/foops)
+51130 51128  gocode        go1.9 (/Users/jbd/bin/gocode)
 ```
-
+The output displays the PID, PPID, the name of the program, the Go version the target program is built, and the location of the associated program.
```

Comment:
Great summary, I think we can further make this information easier to view by making a list like this:

The output displays:
- PID
- PPID
- name of the program
- Go version used to build the program
- The path of the associated program

Reply:
commit updated